src/mrpro/operators/FourierOp.py

```diff
@@ -173,6 +171,11 @@ def forward(self, x: torch.Tensor) -> tuple[torch.Tensor,]:
             shape_nufft_dims = [self._kshape[i] for i in self._nufft_dims]
             x = x.reshape(*permuted_x_shape[: -len(keep_dims)], -1, *shape_nufft_dims)  # -1 is coils
             x = x.permute(*unpermute)
+
+        if len(self._fft_dims):
```

Comment:
no need for len here,
pep8: 
![image](https://github.com/user-attachments/assets/e601c62b-ccae-4562-a6d0-bfae0f55a62a)